api/src/main/java/com/flipkart/poseidon/internal/ParamValidationFilter.java

```diff
@@ -33,7 +33,7 @@
 import java.io.IOException;
 import java.util.*;
 
-import static com.flipkart.poseidon.datasources.util.CallableNameHelper.canonicalName;
+import static com.flipkart.poseidon.helper.CallableNameHelper.canonicalName;
```

Comment:
Update migration notes that CNH is no more available from datasources.util or poseidon.serviceclients.